### ISSUE TITLE
Getting a non-recoverable error with 1.6.17.

### ISSUE THREAD

**Issue by Emmotes** [rimbas/mining-patch-planner#11]
I assume it's because I haven't unlocked Quality Module in this game yet.
```
The mod Mining Patch Planner (1.6.17) caused a non-recoverable error.
Please report this error to the mod author.

Error while running event mining-patch-planner::on_player_cursor_stack_changed (ID 32)
__mining-patch-planner__/mpp/mpp_util.lua:924: attempt to index local 'quality' (a nil value)
stack traceback:
	__mining-patch-planner__/mpp/mpp_util.lua:924: in function 'entity_name_with_quality'
	__mining-patch-planner__/gui/gui.lua:560: in function 'update_miner_selection'
	__mining-patch-planner__/gui/gui.lua:1244: in function 'update_selections'
	__mining-patch-planner__/gui/gui.lua:1267: in function 'show_interface'
	__mining-patch-planner__/control.lua:247: in function <__mining-patch-planner__/control.lua:228>
```

**Comment by rimbas** [comment]
Could I have a save where this happens?

**Comment by Emmotes** [comment]
Sure:
[Test.zip](https://github.com/user-attachments/files/17602126/Test.zip)

There's a quite a few mods so potentially it's a conflict - but this mod was working great with the previous version.

**Comment by rimbas** [comment]
Okay, released update 1.6.18, tell me if it fixes it. I fixed a different bug with migrations clearing blueprints but that caused this bug...

**Comment by Emmotes** [comment]
That's fixed it. No crash when pressing Ctrl+M now.
Thank you. :)
